core/src/main/resources/lib/layout/yui.jelly

```diff
@@ -25,16 +25,9 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
   <st:documentation>
-    Load Yahoo UI module.
-    <st:attribute name="module" use="required">
-      YUI module name to load.
+    Remove this tag, YUI has been removed from Jenkins
+    <st:attribute name="module" deprecated="true">
+      Remove this tag, YUI has been removed from Jenkins
```

Comment:
Nit:

```suggestion
    Do not use this tag, YUI has been removed from Jenkins
    <st:attribute name="module" deprecated="true">
      Do not use this tag, YUI has been removed from Jenkins
```

When first reading this I thought "Why is removal documented instead of just removing it?" -- so prevent anyone else from reading it this way.